ci-operator/step-registry/rosa/terraform/e2e/setup/rosa-terraform-e2e-setup-commands.sh

```diff
@@ -26,13 +26,13 @@ EOF
 export TF_VARS
 
 export ARCHIVE_NAME=account-roles-terraform-archive
-export TF_FOLDER=ci/e2e/account_roles_files
```

Comment:
what it's doing and how is it related to ginkgo/polarion?
(a gentle reminder not to do multiple changes in one PR. it doesn't cost money to split them)

Reply:
Not related.  (ok)